prow/github/client.go

```diff
@@ -408,6 +408,28 @@ func (c *Client) RemoveLabel(org, repo string, number int, label string) error {
 	return err
 }
 
+func (c *Client) AssignIssue(org, repo string, number int, logins []string) error {
+	c.log("AssignIssue", org, repo, number, logins)
+	_, err := c.request(&request{
+		method:      http.MethodPost,
+		path:        fmt.Sprintf("%s/repos/%s/%s/issues/%d/assignees", c.base, org, repo, number),
+		requestBody: map[string][]string{"assignees": logins},
```

Comment:
Yep!

https://developer.github.com/v3/issues/assignees/#add-assignees-to-an-issue and here is me testing it out: https://github.com/kubernetes/test-infra/pull/2113#event-986125801 (I just send a json blob with rmmh multiple times.)